src/CommunityToolkit.Maui/Behaviors/AnimationBehavior.shared.cs

```diff
@@ -24,11 +24,19 @@ public class AnimationBehavior : EventToCommandBehavior
 
 	/// <summary>
 	/// Gets the Command that allows the triggering of the animation.
+	/// 
+	/// NOTE: Apps should not directly set this property, treating it as read only. The setter is only public because
+	/// that's currently needed to make XAML Hot Reload work. Instead, apps should provide a value for this OneWayToSource
+	/// property by creating a binding, in XAML or C#. If done via C# use code like this:
+	/// "behavior.SetBinding(AnimationBehavior.AnimateCommandProperty, nameof(ViewModel.TriggerAnimationCommand));"
 	/// </summary>
 	/// <remarks>
 	/// <see cref="AnimateCommand"/> has a <see cref="Type"/> of Command&lt;CancellationToken&gt; which requires a <see cref="CancellationToken"/> as a CommandParameter. See <see cref="Command{CancellationToken}"/> and <see cref="System.Windows.Input.ICommand.Execute(object)"/> for more information on passing a <see cref="CancellationToken"/> into <see cref="Command{T}"/> as a CommandParameter"
 	/// </remarks>
-	public Command<CancellationToken> AnimateCommand => (Command<CancellationToken>)GetValue(AnimateCommandProperty);
+	public Command<CancellationToken> AnimateCommand {
+		get => (Command<CancellationToken>)GetValue(AnimateCommandProperty);
+		set => SetValue(AnimateCommandProperty, value);
```

Comment:
Apologies for testing something out, I'm on the bus to work so relying on the pipeline builds to tell me whether this will work.

```suggestion
      [Obsolete("Do not use this setter, it only exists to enable XAML Hot reload support in your IDE.")]
		set => SetValue(AnimateCommandProperty, value);
```